pkg/config/config.go

```diff
@@ -238,6 +239,9 @@ type Config struct {
 			Expiry time.Duration `mapstructure:"expiry"`
 			Jitter time.Duration `mapstructure:"jitter"`
 		} `mapstructure:"commit_cache"`
+		Background struct {
+			RateLimit int `mapstructure:"rate_limit"`
```

Comment:
The configure is there but not set (unlimited) as it depends on the use-case.
Plan to leave it as is - so in case there is a problem running the backgound load, we can configure lakeFS with slower rate.